bccsp/pkcs11/pkcs11.go

```diff
@@ -316,6 +337,16 @@ func (csp *impl) generateECKey(curve asn1.ObjectIdentifier, ephemeral bool) (ski
 		listAttrs(p11lib, session, pub)
 	}
 
+	// add key handles to cache
+	csp.p11Keys.set(
+		hex.EncodeToString(ski),
```

Comment:
nit: we can embed in the implementation instead of at the call site since the key at this level is the byte slice. This keeps the machinery for keys a details of the cache.

If we extend this to public/private handles, the cache can then generate keys appropriately based on context.

Reply:
Done